simtools/visualization/visualize.py

```diff
@@ -748,9 +748,10 @@ def _kernel_plot_2D_photons(corsika_output_instance, property_name, log_z=False)
         "time_altitude": "Altitude of emission (km)",
         "num_photons_per_telescope": "Telescope index",
     }
+    _logger = logging.getLogger(__name__)
```

Comment:
Please note that this is how we are doing in other places as well (see image below). The line `logging.getLogger(__name__) ` appears only in 3 functions in this file.

<img width="954" alt="Screenshot 2023-06-20 at 15 06 00" src="https://github.com/gammasim/gammasim-tools/assets/50831535/9af5b92b-51bc-47e5-b5c1-47d6d3216bec">

I don't understand from which import you want to get the logger. The function here in visualize might be used by different objects, which could have different loggers. I am in favour of keeping the logger in the original function.

Reply:
I meant to move this call to get the logger outside of the functions so that we need to do it only once. I just tested it and it should work exactly the same as it is now. It has the advantage that we remove this duplicate lines. I committed this change to this branch, see if you agree with it and if the tests run well. If you do not agree just revert back.

Reply:
ok, I agree with that. I would have a hard time agreeing to import a logger. thanks for the commit!